src/Snapshooter.Json/Snapshooter.Json.csproj

```diff
@@ -1,7 +1,7 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>netstandard2.1</TargetFramework>
+    <TargetFrameworks>netstandard2.0; netstandard2.1</TargetFrameworks>
```

Comment:
netstandard2.0 would be enough. However, thats okay for me.

Reply:
i willl change it to netstandard2.0

Reply:
Perfect, thx